modules/vaos/spec/factories/va_appointment_requests.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :va_appointment_request, class: 'VAOS::VAAppointmentRequest' do
+    appointment_type  { 'Mental Health' }
+    visit_type  { 'Office Visit' }
+
+    facility do
+      {
+        facility_address: '1201 Broad Block Blvd',
+        facility_city: 'Richmond',
+        facility_state: 'VA',
+        facility_code: '652',
+        facility_name: 'Richmond VAMC',
+        facility_parent_site_code: '688',
+        facility_type: 'VAMC'
+      }
+    end
+
+    email { 'test@va.gov' }
+    phone_number  { '+1 (555) 555-5555 x55.55' }
```

Comment:
dots are allowed in the phone extension?

Reply:
Yes. It's not a typo.